src/orquestra/sdk/_ray/_build_workflow.py

```diff
@@ -214,7 +214,14 @@ def _ray_remote(*inner_args, **inner_kwargs):
                     deserialize=serialization,
                 )
 
-                wrapped_return = wrapped(*inner_args, **inner_kwargs)
+                try:
+                    wrapped_return = wrapped(*inner_args, **inner_kwargs)
+                except SystemError as e:
+                    raise EnvironmentError(
+                        "Could not deserialise data. "
+                        "This may be due to a python version mismatch. "
+                        "Please try updating your python version."
```

Comment:
```suggestion
                        "This may be due to a Python version mismatch. "
                        "Please try updating your Python version."
```

Reply:
Actually, the action here isn't too helpful. Can we tell the user what version of Python they should install?